ztp/siteconfig-generator/siteConfig/siteConfigHelper.go

```diff
@@ -54,6 +54,8 @@ func MergeManifests(individualMachineConfigs map[string]interface{}, doNotMerge
 		merged.SetName(fmt.Sprintf("%s-%s", McName, roleName))
 		merged.ObjectMeta.Labels = make(map[string]string)
 		merged.ObjectMeta.Labels[machineconfigv1.MachineConfigRoleLabelKey] = roleName
+		merged.ObjectMeta.Annotations = make(map[string]string)
+		merged.ObjectMeta.Annotations[ZtpAnnotation] = ZtpAnnotationDefaultValue
```

Comment:
I know it would only be used in one place for now, but maybe it's worth adding a `addZtpAnnotationToObject(object)` and pass in `merged`?

Reply:
I don't see it as added value.